assets/js/theme/common/carousel/utils/tooltipSetup.js

```diff
@@ -0,0 +1,30 @@
+const carouselTooltipClass = 'carousel-tooltip';
+const carouselTooltip = `<span class="${carouselTooltipClass}"></span>`;
+
+const setupTooltipAriaLabel = ($node) => {
+    const $existedTooltip = $node.find(`.${carouselTooltipClass}`);
+
+    if ($existedTooltip.length) {
+        $existedTooltip.attr('aria-label', $node.attr('aria-label'));
+    } else {
+        const $tooltip = $(carouselTooltip).attr('aria-label', $node.attr('aria-label'));
+        $node.append($tooltip);
+    }
+};
+
+const arrows = (...arrowNodes) => {
```

Comment:
Could we add more specific name for this function?

Reply:
No problem - setupArrowTooltiops?

Reply:
👍